tests/Amazon.CloudWatch.EMF.Tests/Environment/EnvironmentProviderTests.cs

```diff
@@ -36,6 +37,7 @@ public void ResolveEnvironment_ReturnsLambdaEnvironment()
             configuration.EnvironmentOverride.Returns(Environments.Lambda);
             var resourceFetcher = fixture.Create<IResourceFetcher>();
             var environmentProvider = new EnvironmentProvider(configuration, resourceFetcher);
+            environmentProvider.CleanResolvedEnvironment();
```

Comment:
You can use `IDisposable` to clean up the environment instead of copying it to each method.

Reply:
Thanks for the suggestion. Implemented it [here](https://github.com/awslabs/aws-embedded-metrics-dotnet/pull/46/commits/bb8c62ddcd79494c9e9877f5ccd1b852d38546ef).